FraudProtection/src/main/java/com/paypal/android/fraudprotection/PayPalDataCollector.kt

```diff
@@ -35,24 +35,46 @@ class PayPalDataCollector internal constructor(
      *
      * @return clientMetadataId Your server will send this to PayPal
      */
+    @Deprecated("This method is no longer supported.")
```

Comment:
Can we link to the new method in these deprecation methods? like "Use [link to method] instead"

Reply:
Kind of, there's something called `ReplaceWith` but the docs weren't very clear. [This article](https://dev.to/mreichelt/the-hidden-kotlin-gem-you-didn-t-think-you-ll-love-deprecations-with-replacewith-3blo) helps explain it a bit, but our use-case may be more complex than re-arranging a few parameters since we're introducing a new request object to the API.